src/main/kotlin/me/spradling/gift/core/database/aws/RDSGiftCommitStorageClient.kt

```diff
@@ -54,21 +55,43 @@ class RDSGiftCommitStorageClient @JsonCreator constructor(
     val accounts = mutableListOf<Account>()
     var resultSet = connection.createStatement().executeQuery(getQuery)
 
-    while(resultSet.next()) {
+    while (resultSet.next()) {
       accounts.add(Account(resultSet.getString("accountId"),
-                           resultSet.getString("groupId"),
-                           resultSet.getString("firstName"),
-                           resultSet.getString("lastName"),
-                           resultSet.getString("email"),
-                           resultSet.getString("password")))
+          resultSet.getString("groupId"),
+          resultSet.getString("firstName"),
+          resultSet.getString("lastName"),
+          resultSet.getString("email"),
+          resultSet.getString("password")))
     }
 
     if (accounts.isEmpty()) {
       throw ResourceNotFoundException()
     }
 
     return Future.succeededFuture(accounts[0])
+  }
+
+  override fun getAccounts(limit: Int): Future<List<Account>> {
+    val getQuery = "SELECT * from $accountTable LIMIT $limit"
```

Comment:
what happens if the user doesn't specify a `limit`?

Reply:
If I'm not mistaken, that behavior is prevented by the `if (limit.isNotBlank())` check in the `GetAccountsHandler`